Dockerfile

```diff
@@ -41,6 +41,10 @@ RUN git clone -c advice.detachedHead=false https://github.com/opencontainers/run
 RUN make static BUILDTAGS="seccomp apparmor" && \
     cp runc /usr/bin/
 
+FROM gobase AS rootlesskit
+ENV GO111MODULE=on
```

Comment:
Why set this envvar? I don't think you're in the GOPATH and there's no `go.mod`, right?

Read this cool article last week: https://dev.to/maelvls/why-is-go111module-everywhere-and-everything-about-go-modules-24k

Reply:
Hmm
```
Step 18/40 : RUN go get github.com/rootless-containers/rootlesskit/cmd/rootlesskit@v0.10.0
 ---> Running in bda58396893f
go: cannot use path@version syntax in GOPATH mode
```